add a note about list params to the changelog file (#5349)
firebase/firebase-tools: CHANGELOG.md

> - Support for string list typed parameters in deployment, when the SDK starts emitting them (#5137)

Reviewer: ```suggestion
- Support for string list typed parameters in functions deployment (#5137)
```
